doc/bootstrap.rst

```diff
@@ -6,22 +6,35 @@ A new Ceph cluster is deployed by bootstrapping a cluster on a single
 node, and then adding additional nodes and daemons via the CLI or GUI
 dashboard.
 
+In the example here, we will install a basic three-node cluster. Each
+node will be identified by its prompt. For example, "[monitor 1]"
+identifies the first monitor, "[monitor 2]" identifies the second
+monitor, and "[monitor 3]" identifies the third monitor. This
+information is provided in order to make clear which commands
+should be issued on which systems.
+
+"[any monitor]" identifies any Ceph node, and in the context
```

Comment:
"any monitor" or "any node"? Reading the paragraph, I expect you want to write "node".

Reply:
This is... unclear to me at this time. The idea here is that within about a week there will exist a Getting Started Guide that explains how to set up a three-node Ceph cluster, and that anyone with access to three computers and the ability to read English and type commands can get a three-node basic Ceph cluster out of it. I was using "monitor" here because all three of the initial nodes will be made into monitors. I see your point, though. In the absence of the full context of the Getting Started Guide, "monitor" is less correct here than "node" would be. I'll have to mull it over for a few hours, but I suspect that I'll be making this suggested change.

Reply:
I have made this change.